.github/workflows/target-creation.yml

```diff
@@ -0,0 +1,41 @@
+name: CI - Upload config.h to API
+
+on:
+  push:
+    branches:
+    - 'master'
```

Comment:
What about adding something similar to the configurator to execute this only when needed?
https://github.com/betaflight/betaflight-configurator/blob/43e080b81fcfea243e488c5dd0fa1bc2c9464812/.github/workflows/translations-upload.yml#L11

Reply:
Done :)